### DISCUSSION TITLE
Conflict with navigator plugin?

### DISCUSSION THREAD

**Discussion by danraymond** [NicolasPetton/Notebook#20] [General]
@NicolasPetton 

I use BTC's navigator plugin. I have noticed that if I use the search bar in your theme, whilst it searches and gives results, when I click on a link to open the selected result tiddler nothing happens. If I disable the navigator plugin all returns to normal.
Woud appreciate some help wkiht this as I am trying to move to your notebook them as my main theme.

**Comment by NicolasPetton** [comment]
@danraymond could you send me a link to the navigator plugin? I'll have a look.

**Reply by kookma** [reply]
I think @BurningTreeC can explain this! but I think navigator is a rather old plugin.

**Comment by BurningTreeC** [comment]
@danraymond @NicolasPetton @kookma,

the navigator plugin is rather old and overwrites many shadow tiddlers. I can't - as I already stated before - assure compatibility with newer tiddlywiki versions and different themes/plugins etc.

Some features of the navigator plugin are now part of the tiddlywiki core and some may be part in the future, like Story-River navigation using keyboard shortcuts.

**Reply by danraymond** [reply]
@NicolasPetton I fully understand and thank you for looking at it. Will be great when keyboard navigation is available fully in the story river. I like BTC's implementation but as I prefer your theme I wil look to the future in keyboard navigation and abandon his plugin for now. Thank you again.
